pkg/rest/request/request.go

```diff
@@ -54,6 +54,10 @@ func (c Client) Send(ctx context.Context, route, method string, body, responsePt
 		req.Header[key] = values
 	}
 
+	if c.host != "" {
+		req.Host = c.host
+	}
+
```

Comment:
since both are strings their default values will be "".

Meaning that if `c.Host == "" ` it means that it will attach `""` to an req.Host that already has the value `""`

So in my eyes there is no problem to do just `req.Host = c.host`, since if its empty the value of req.Host will not leave the default. What i'm saying is that there is no need for the `if`, maybe just adding a comment to make it clear on what `req.Host = c.host` is doing